airbyte-integrations/connectors/source-pokeapi/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-standard-source-test-file'
+}
+
+airbytePython {
+    moduleDirectory 'source_pokeapi'
+}
+
+airbyteStandardSourceTestFile {
+    // For more information on standard source tests, see https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/testing-connectors
+
+    // All these input paths must live inside this connector's directory (or subdirectories)
+    // TODO update the spec JSON file
+    specPath = "source_pokeapi/spec.json"
+
+    // configPath points to a config file which matches the spec.json supplied above. secrets/ is gitignored by default, so place your config file
+    // there (in case it contains any credentials)
+    // TODO update the config file to contain actual credentials
+    configPath = "secrets/config.json"
```

Comment:
you'll need to change this to `sample_files/config.json`

Reply:
Missed this comment for some reason! Done.